Bottom: eb966472e0c0eacea9023dc99b49de9ac1858b9a
Top:    3d7a6fcb359026efffafa848528e8741f82a1d68
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2014-06-29 17:29:00 +0100

Refresh of tag-vnodes-with-details-of

---

diff --git a/src/polygon.c b/src/polygon.c
index 7b390a3..fbc0c29 100644
--- a/src/polygon.c
+++ b/src/polygon.c
@@ -105,11 +105,11 @@ dicer output is used for HIDs which cannot render things with holes
 #endif
 
 
-#undef DEBUG_CIRCSEGS
+#define DEBUG_CIRCSEGS
 
 #define ROUND(x) ((long)(((x) >= 0 ? (x) + 0.5  : (x) - 0.5)))
 
-#define UNSUBTRACT_BLOAT MIL_TO_COORD (100)
+#define UNSUBTRACT_BLOAT MIL_TO_COORD (0.1)
 #define SUBTRACT_PIN_VIA_BATCH_SIZE 1 /*100*/
 #define SUBTRACT_LINE_BATCH_SIZE 1 /*20*/
